drizzlepac/alignimages.py

```diff
@@ -0,0 +1,1075 @@
+#!/usr/bin/env python
+
+"""This script is a modernized implementation of tweakreg.
+
+"""
+
+import datetime
+from astropy.io import fits
+from astropy.table import Table
+from astropy.coordinates import SkyCoord, Angle
+from astropy import units as u
+from collections import OrderedDict
+from drizzlepac import updatehdr
+import glob
+import math
+import numpy as np
+import os
+import pdb
+import pickle
+from stsci.tools import fileutil
+from stwcs.wcsutil import headerlet
+from stwcs.wcsutil import HSTWCS
+import sys
+import tweakwcs
+try:
```

Comment:
@mdlpstsci This `try-except` block does not do anything. Also why imports below are absolute?